product_active_propagate/readme/INSTALL.rst

```diff
@@ -0,0 +1,34 @@
+If you are installing this module on a database with existing data, you could want
+to execute the two following SQL requests, to fix some inconsistent data.
+
+
+* Disable templates that do not have a single variant active
+
+.. code-block:: sql
+
+    UPDATE product_template pt
+    SET active = false
+    AND id in (
+        SELECT pt.id
+        FROM product_template pt
+        INNER JOIN product_product pp on pp.product_tmpl_id = pt.id
+        WHERE pt.active = true
+        GROUP BY pt.id
+        HAVING sum((pp.active)::int) = 0
+    );
```

Comment:
```suggestion
    UPDATE product_template pt
    SET active = false
    WHERE id in (
        SELECT pt.id
        FROM product_template pt
        INNER JOIN product_product pp on pp.product_tmpl_id = pt.id
        WHERE pt.active = true
        GROUP BY pt.id
        HAVING sum((pp.active)::int) = 0
    );
```

Reply:
I don't know if this depends on version of postgresql but in my local installation the current query disable all the templates, with this change it works, disabling only the templates that do not have a single variant active.

Reply:
how ! thanks a lot ! my bad. I fixed the request.